x/stakeibc/keeper/icacallbacks_delegate.go

```diff
@@ -63,7 +62,7 @@ func DelegateCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, ack
 	txMsgData, err := icacallbacks.GetTxMsgData(ctx, *ack, k.Logger(ctx))
 	if err != nil {
 		k.Logger(ctx).Error(fmt.Sprintf("failed to fetch txMsgData, packet %v", packet))
-		return sdkerrors.Wrap(icacallbackstypes.ErrTxMsgData, err.Error())
+		return fmt.Errorf(icacallbackstypes.ErrTxMsgData.Error(), err.Error())
 	}
```

Comment:
@hoangdv2429: I think it should be: `return fmt.Errorf("%s: %s", err.Error(), icacallbackstypes.ErrTxMsgData.Error())`